try:
    x = 1 / 0
except Exception as e:
    print(e)

x = 3 + 2
print x